tests/Feature/Api/V1/UserWantToPlayListTest.php

```diff
@@ -0,0 +1,297 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use App\Community\Enums\UserGameListType;
+use App\Community\Enums\UserRelationship;
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use App\Models\UserGameListEntry;
+use App\Models\UserRelation;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class UserWantToPlayListTest extends TestCase
+{
+    use RefreshDatabase;
+
+    protected User $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        /** @var User $user */
+        $user = User::factory()->create(['User' => 'myUser']);
+        $this->user = $user;
+    }
+
+    protected function apiUrl(string $method, array $params = []): string
+    {
+        $params = array_merge(['y' => $this->user->APIKey], $params);
+
+        return sprintf('API/API_%s.php?%s', $method, http_build_query($params));
+    }
```

Comment:
I am overriding the `BootstrapV1Api` methods here to ensure that `myUser` is the one that is making the API calls.